src/views/CurrencyExchange.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <div>
+      <div class="text-center">
+        <h1 class="display-7">Currency Exchange</h1>
+      </div>
+      
+      <div class="text-center">
+        <div class="btn-group">
+          <select v-model="startingCurrency" class="form-select form-select-lg mb-3" style="width:300px">
+            <option value="" disabled>Select Starting Currency</option>
+            <option v-for="(currency, index) in currencies" :key="index" :value="currency">{{ currency }}</option>
+          </select>
+          &ensp;&ensp;
+          <select v-model="endingCurrency" class="form-select form-select-lg mb-3" style="width:300px">
+            <option value="" disabled>Select Ending Currency</option>
+            <option v-for="(currency, index) in currencies" :key="index" :value="currency">{{ currency }}</option>
+          </select>
+        </div>
+      </div>
+      
+      <div class="text-center">
+        <div class="btn-group">
+          <div class="input-group mb-3">
+            <span class="input-group-text">$</span>
+            <input v-model.number="amount" type="number" class="form-control" aria-label="Amount (to the nearest dollar)">
+            <span class="input-group-text">.00</span>
+          </div>
+          &ensp;&ensp;
+          <div class="input-group mb-3">
+            <input :value="convertedAmount" type="text" class="form-control" aria-label="Amount (to the nearest dollar)" disabled readonly>
+            <span class="input-group-text">.00</span>
+          </div>
+        </div>
+      </div>
+  
+      <div class="text-center">
+        <button @click="convertCurrency" type="button" class="btn btn-primary"
+          style="--bs-btn-padding-y: .25rem; --bs-btn-padding-x: 7.5rem; --bs-btn-font-size: 2rem;">
+          Convert
+        </button>
+      </div>
+    </div>
+  </template>
+  
+  <script>
+  import axios from 'axios';
+  
+  export default {
+    data() {
+      return {
+        startingCurrency: '',
+        endingCurrency: '',
+        amount: 0,
+        conversionRate: null,
+        apiKey: 'fca_live_0nD3HHGKOsJYluJOoEnHCVs7kg5LQEbyfrARpnLV',
+        currencies : [ "AUD","BGN","BRL","CAD","CHF","CNY","CZK","DKK","EUR","GBP","HKD","HRK","HUF","IDR","ILS","INR","ISK","JPY","KRW","MXN","MYR","NOK","NZD","PHP","PLN","RON","RUB","SEK","SGD","THB","TRY","USD","ZAR"]
```

Comment:
I wonder if there's a way in the future to get these from the api so they don't have to be hardcoded? It's fine for now though.